pkgs/development/tools/github-changelog-generator/shell.nix

```diff
@@ -0,0 +1,16 @@
+# Env to update Gemfile.lock / gemset.nix
+with import <nixpkgs> {};
+stdenv.mkDerivation {
+  name = "env";
+  buildInputs = [
+    ruby.devEnv
+    git
+    sqlite
+    libpcap
+    postgresql
+    libxml2
+    libxslt
+    pkgconfig
+    bundix
+  ];
+}
```

Comment:
I don’t understand the purpose of this file — you can update the lock file without actually building and installing the gems with `bundle lock --update`.